week1/웨비의 사진관/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>봄여름가을겨울</title>
+  </head>
+  <body>
+    <header>계절 별 소중한 추억들 😊</header>
+    <section id="preview">
+      <div class="gallery">
+        <img src="./assets/img/preview/pr1.jpg" alt="롯데월드 후문" />
+        <img src="./assets/img/preview/pr2.jpg" alt="밤 벚꽃 가까이" />
+        <img src="./assets/img/preview/pr3.jpg" alt="장미와 함께 하트 사진" />
+        <img src="./assets/img/preview/pr4.jpg" alt="여름 수국" />
+        <img src="./assets/img/preview/pr5.jpg" alt="가을 갈대" />
+        <img src="./assets/img/preview/pr6.jpg" alt="가을 단풍" />
+        <img src="./assets/img/preview/pr7.jpg" alt="크리스마스 왕보리" />
+        <img src="./assets/img/preview/pr8.jpg" alt="크리스마스 트리" />
+      </div>
+    </section>
+    <div class="seperate-line"></div>
+    <nav>
+      <span>클릭 시 이동</span>
+      <div class="summary">
+        <a href="#spring">봄🌸</a>
+        <a href="#summer">여름🌿</a>
+        <a href="#autumn">가을🍁</a>
+        <a href="#winter">겨울☃️</a>
+      </div>
+    </nav>
+    <div class="seperate-line"></div>
+    <section id="spring">
+      <div class="title">봄🌸</div>
```

Comment:
여기도 div보다는
```suggestion
      <header class="title"><h2>봄🌸</h2></div>
```
이런식으로 더 시맨틱하게 작성해볼까요?!

Reply:
시멘틱 태그에 대한 부분을 다시한번 더 알아가네요 ! 부족한 html 실력을 채울 수 있는거 같습니다🔥